eng/Versions.props

```diff
@@ -12,7 +12,7 @@
          as for the patches and revisions should be manually updated per assembly if it is serviced. -->
     <!-- Changing assembly versions without changing TFM lead to broken packages.
          This shoud be incremented after introducing the 6.0 TFM. -->
-    <AssemblyVersion>5.0.0.0</AssemblyVersion>
+    <AssemblyVersion>6.0.0.0</AssemblyVersion>
```

Comment:
I remember there were some assemblies that overrode this that couldn't be versioned as 5.0.0.0, right? Do we need to bump those?

Reply:
They still need to be pinned to match the versions on desktop

Reply:
I think @safern was asking about things which weren't able to use the product version.  Here's one: https://github.com/dotnet/runtime/blob/466e4b78e4a06a73ea6cbe24324c6256fc546f58/src/libraries/Microsoft.VisualBasic.Core/Directory.Build.props#L6

Reply:
Maybe this one too:
https://github.com/dotnet/runtime/blob/466e4b78e4a06a73ea6cbe24324c6256fc546f58/src/libraries/System.Reflection.Context/Directory.Build.props#L4

Reply:
What's the reason that System.Reflection.Context wasn't updated to 5.0.0?

Reply:
Hmm I assume it's the same reason as for System.ValueTuple?

>     <!-- Must match version supported by frameworks which support 4.0.* inbox.
         Can be removed when API is added and this assembly is versioned to 4.1.* -->

Reply:
If it's a 1:1 assembly with desktop and exposes a netstandard ref we need to make sure the reference assembly is pinned at 4.0.0.0 and implementation can still rev.

Reply:
Yes, the reference assembly is pinned to 4.0.0.0, so I'm wondering why the src wasn't bumped to 5.0.0.0.

Reply:
yes, they need to have similar major and minor digits.